core/client/fs/src/main/java/alluxio/client/block/AlluxioBlockStore.java

```diff
@@ -131,28 +133,62 @@ public BlockInfo getInfo(long blockId) throws IOException {
    * @return a stream which reads from the beginning of the block
    */
   public BlockInStream getInStream(long blockId, InStreamOptions options) throws IOException {
+    return getInStream(blockId, options, ImmutableMap.of());
+  }
+
+  /**
+   * Gets a stream to read the data of a block. This method is primarily responsible for
```

Comment:
javadoc the difference of this overriding method from the above one?

Reply:
Good suggestion. Updated.